### PR TITLE
WorldMrg get object generics

### PR DESCRIPTION
**Pull request by LRFalk01**

### CODE REVIEW

File: GameServerScripts/GameServerScripts.csproj [hunk 1]
```diff
@@ -149,6 +149,9 @@
     <Compile Include="globals.cs">
       <SubType>Code</SubType>
     </Compile>
+    <Compile Include="quests\Albion\GreetingsArmsman.cs" />
+    <Compile Include="quests\Albion\GreetingsPaladin.cs" />
+    <Compile Include="quests\Hibernia\BonesToBlades.cs" />
```
Comment (benfalk) [inline GameServerScripts/GameServerScripts.csproj:154]: What are the changes to this file fore?

File: GameServerScripts/GameServerScripts.csproj.user [hunk 2]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (benfalk) [inline GameServerScripts/GameServerScripts.csproj.user]: Another file I'm curious about

**Review by benfalk — COMMENTED** [review]
Curious about the two files I made reference to

File: GameServerScripts/GameServerScripts.csproj (hunk 1 above)
Comment (LRFalk01) [reply GameServerScripts/GameServerScripts.csproj:154]: Files were not in project but should have been.

File: GameServerScripts/GameServerScripts.csproj.user (hunk 2 above)
Comment (LRFalk01) [reply GameServerScripts/GameServerScripts.csproj.user]: File should be added to .gitignore

**Review by benfalk — APPROVED** [review]

**Pull request merged**